minecraft_servers/phoenix/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "phoenix",
+  "nice_name": "Phoenix",
+  "direct_ip": "rainbowfurry.com",
+  "server_wildcards": [
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://rainbowfurry.com",
+    "web_shop": "https://rainbowfurry.com"
+  },
+  "gamemodes": {
+    "clashofclans": {
+      "name": "Clash of Clans",
+      "command": "/quickjoin CoM",
+      "url": "https://www.rainbowfurry.com",
```

Comment:
remove if you don't have a dedicated page for this gamemode

Reply:
Changed the URL for gamemode